src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -127,6 +127,12 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	override func viewDidAppear(_ animated: Bool) {
 		super.viewDidAppear(animated)
 
+		#if DEBUG
+		if isUITesting && UserDefaults.standard.string(forKey: "showTestResultCards") == "YES" {
+			tableView.scrollToRow(at: IndexPath(row: 1, section: 1), at: .top, animated: false)
```

Comment:
Is this needed? Why is the UI-Test process not "scrolling" to this position on its own?

Reply:
Yes, it is needed. For some reason, wait for existence is not working, and the required screenshot like both cards on the same view can not be achieved by swipeup/swipedown, hence we need to come up with this.

Reply:
Unfortunately, we don’t have the scroll to or some similar function while UI testing.